core/models.py

```diff
@@ -9,7 +12,15 @@
 from django.utils import timezone
 from django.utils.text import format_lazy
 from django.utils.translation import gettext_lazy as _
+from django.utils.timezone import now
+from django.core.validators import RegexValidator
 
+import random
+
+from taggit.managers import TaggableManager
+from taggit.models import TagBase, GenericTaggedItemBase, TaggedItemBase
+
+random.seed(time.time())
```

Comment:
Instead of the extra import lets just use `now().timestamp()` to seed.

Reply:
Also it is necessary to seed here? Any reason _not_ to see in `random_color` specifically?

Reply:
Actually, it was just "muscle memory" - probaby from C. Default seed is ok as well, I think :)